src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -79,6 +80,22 @@ class StepFlowPickHandler extends LitElement {
             `
         )}
       </div>
+      <p style=${styleMap({ textAlign: `center` })}>
```

Comment:
Don't use a `styleMap` for static styles, they are for dynamic styles. Just add this styles to the `static get styles()`